Dockerfile

```diff
@@ -6,7 +6,7 @@ ENV DEBIAN_FRONTEND=noninteractive
 RUN apt-get update &&  apt-get -qq install -y \
       ruby ruby-dev \
       libpq-dev postgresql-client \
-      mysql-client \
+      libmysqlclient-dev mysql-client \
```

Comment:
I think we do for the `wait for it` script no? How would we check if the database is ready otherwise?